### ISSUE TITLE
Plugins: Github link missing screen reader text

### ISSUE THREAD

**Issue by marcysutton** [gatsbyjs/gatsby#11633]
## Description

The link to a plugin's source on Github uses an SVG icon and it is missing a text alternative. If a screen reader user navigates to this page and lands on that link, they won't know what it's for.

### Steps to reproduce

- Go to a plugin page like https://www.gatsbyjs.org/packages/gatsby-image/
- Above the header for the plugin (in this case, "gatsby-image"), inspect the Github icon link. 
- You can confirm this with the Chrome accessibility inspector: https://developers.google.com/web/tools/chrome-devtools/accessibility/reference#aria

### Expected result

The link should have a text alternative for screen reader users. This can be done in a few different ways:
1. Use an `aria-label` on the link.
2. Add a `<title>` in the SVG.

The text should say something like "gatsby-image plugin source on Github" or "Github source code for gatsby-image plugin"

Another good thing would be to add `focusable: false` to the SVG so it can't steal focus in some browsers/assistive technologies.

### Actual result

There is no text alternative in the Github link or SVG.

![gatsby-image inspected in Chrome a11y iinspector, missing a text alternative](https://user-images.githubusercontent.com/1045233/52442570-52aeca00-2ad8-11e9-837e-47cf44d81526.png)

**Comment by marcysutton** [comment]
Fixed with #11647.
